translations/en.i18n.json

```diff
@@ -20,7 +20,7 @@
       "attachTranscriptText": "Transcript popped out",
       "attachTranscriptButton": "Bring it back",
       "attachTranscript": "Bring Transcript back",
-      "detachTranscript": "Popout Transcript",
+      "detachTranscript": "Popout transcript",
```

Comment:
what about the other languages ?

Reply:
there ticket for it https://kaltura.atlassian.net/browse/FEC-14008